subsys/net/lib/nrf_cloud/src/nrf_cloud_pgps.c

```diff
@@ -188,8 +189,8 @@ static int validate_prediction(const struct nrf_cloud_pgps_prediction *p,
 
 	if ((gps_sec < pred_sec) || (gps_sec > end_sec)) {
 		LOG_ERR("prediction does not contain desired time; "
-			"start:%lld, cur:%lld, end:%lld",
-			pred_sec, gps_sec, end_sec);
+			"start:%d, cur:%d, end:%d",
+			(int32_t)pred_sec, (int32_t)gps_sec, (int32_t)end_sec);
```

Comment:
in general, why are these gps time values signed?

Reply:
That's because the underlying `date_time_now()` function from the date time library returns the current time as a signed int64_t.

Reply:
I could rewrite it (in many places) to be uint32_t, but sometimes delta times are needed, so it was cleaner to leave it as signed.